README.md

```diff
@@ -57,3 +57,4 @@ juju run-action <unit> remove-default-gw --wait
 ## Relations
 
 - **lte-vepc**: LTE VEPC Interface. Shares enodeB's MME address.
+- **lte-core**: LTE Core Interface. Shares enodeB'.
```

Comment:
Yes, I didn't remove it yet. Just in case someone from osm was using it.

Reply:
I think we should remove it to avoid confusion. Also descriptions should be changed - now the first one suggests that the eNB has an MME unit and the second one looks like the `lte-core` was sharing the eNB... ;)

Reply:
Well, we can ask (as you did) and give them a heads-up, but in the end, IMO, whoever uses this charm will have to adjust.

Reply:
Removed.